dll/win32/shell32/folders/CFSFolder.cpp

```diff
@@ -1598,27 +1598,28 @@ HRESULT WINAPI CFSFolder::GetDetailsOf(PCUITEMID_CHILD pidl,
     else
     {
         hr = S_OK;
-        psd->str.uType = STRRET_CSTR;
+        psd->str.uType = STRRET_WSTR;
+        psd->str.pOleStr = (LPWSTR)CoTaskMemAlloc(MAX_PATH * sizeof(WCHAR));
```

Comment:
Where does this `pOleStr` get freed ?

Reply:
The caller has to free it.

Reply:
excerpt from the chat:
`rfb:` you merged, what about that discussed assumed memory-leak in there?
`Katayama:` the caller should free it
`rfb:` yeah sure. don't we have any callers of that function already? I would assume so, because otherwise you most likely wouldn't have had to touch the func to fix the bug with the displaying. that makes me think that the func very likely is called already somewhere.
`Katayama:` `StrRetTo... `functions do handle it.
`rfb:` and those are the only callers of CFSFolder::GetDetailsOf() until now?

(no further reply yet, should be double-checked)